db/blip_handler.go

```diff
@@ -1311,6 +1327,13 @@ func (bh *blipHandler) sendGetAttachment(sender *blip.Sender, docID string, name
 	if !bh.sendBLIPMessage(sender, outrq) {
 		return nil, ErrClosedBLIPSender
 	}
+	messageCPUtime := time.Since(startTime).Milliseconds()
```

Comment:
this comes from `processRev` if I'm ready this code correctly, and it's worth documenting how you can get here.

This will be run while `processRev` handler runs, but it the handler in `register` is running for the revision. I think this is fine.

Reply:
Good point this I guess is already being measured in the handler code, I feel like this should be removed then? But I guess its new messge being sent so should still be counted. Ill add the comments to expalin this. Good spot.